rotkehlchen/chain/evm/tokens.py

```diff
@@ -64,7 +64,7 @@
 # tokens the benchmark will probably have to run again.
 
 
-OTHER_MAX_TOKEN_CHUNK_LENGTH = 590
+OTHER_MAX_TOKEN_CHUNK_LENGTH = 460
```

Comment:
Can you update the benchmark above and explain why this value is chosen? As the benchmark and the entire comment is for a completely different contract. It now needs to change.